docs/src/contribute/governance.md

```diff
@@ -32,12 +32,17 @@ Website Team Members are community members who have shown that they are committe
 
  Website Team Members:
 
+* Are expected to work at least one hour per week triaging issues and reviewing pull requests.
+* Are expected to work at least two hours total per week on ESLint.
+* May invoice for the hours they spend working on ESLint at a rate of $50 USD per hour.
+* Are expected to check in on the `#team` Discord channel once per week day for team updates.
```

Comment:
This assumes folks never take any week days off. What kind of time off is acceptable?

Proposal: maybe rephrase this to say something like "per _working_ day", and add a list item about expectations of working most weeks in a month?

_(here and for committers)_

Reply:
We don't have any specific expectations around working weeks in a month. Would that be helpful for you? 

I'm not sure what the definition of "working day" is, so I fear that would introduce more confusion. Maybe "week days (excluding local holidays and other time off)"?

The point I'm trying to make here is that we need people active during the week but not on the weekends (unless you want to).

Maybe we can update the bullet about letting us know about time off with something like "You are free to take time off whenever you want, just post in `#team` if you're going to be away for more than a couple of days."?

Reply:
Yeah that makes sense. I think it's generally hard to be precise here. 👍 from me on either or both of those changes.

Reply:
Okay, see if this latest wording helps.